src/functions/template-tags/general.php

```diff
@@ -198,6 +198,12 @@ function tribe_is_ajax_view_request( $view = false ) {
 	 * @return bool true if this post is an Event post type
 	 */
 	function tribe_is_event( $postId = null ) {
+		/**
+		 * Filter: 'tribe_is_event'.
+		 *
+		 * @param bool $is_event
+		 * @param int $postId
```

Comment:
This one doesn't seem to be new right? Just checking if we should add the `@since`

Reply:
Yes that's right this one is not part of tnis PR I just added some dockblock for the filer this PR.